### ISSUE TITLE
[WIN] Moving symlinks from temporary to install folder fails

### ISSUE THREAD

**Issue by bezin** [contao/contao-manager#99]
Hello there,

when running CM on Windows, the installation task fails, when it tries to move the system/logs link from the temporary to the install folder. More precisely the actual unlink() fails due to a lack of permission.

```
Chained exception: Warning: unlink(C:\development\contao\contao-manager\test-dir\install-5946c150918a0\system\logs): Permission denied
#0 C:\development\contao\contao-manager\vendor\tenside\core\src\Task\Composer\InstallTask.php(191): Tenside\Core\Task\Composer\InstallTask->moveFile(Object(Symfony\Component\Finder\SplFileInfo), 'C:\\development\\...', false, Object(Composer\IO\ConsoleIO))
#1 C:\development\contao\contao-manager\vendor\tenside\core\src\Task\Composer\InstallTask.php(109): Tenside\Core\Task\Composer\InstallTask->moveFiles()
#2 C:\development\contao\contao-manager\vendor\tenside\core\src\Task\Task.php(173): Tenside\Core\Task\Composer\InstallTask->doPerform()
#3 C:\development\contao\contao-manager\vendor\tenside\core\src\Task\Runner.php(89): Tenside\Core\Task\Task->perform('C:\\development\\...')
#4 C:\development\contao\contao-manager\vendor\tenside\core-bundle\src\Command\RunTaskCommand.php(82): Tenside\Core\Task\Runner->run('C:\\development\\...')
#5 C:\development\contao\contao-manager\vendor\symfony\console\Command\Command.php(264): Tenside\CoreBundle\Command\RunTaskCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 C:\development\contao\contao-manager\vendor\tenside\core-bundle\src\Command\RunTaskCommand.php(61): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 C:\development\contao\contao-manager\vendor\symfony\console\Application.php(887): Tenside\CoreBundle\Command\RunTaskCommand->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 C:\development\contao\contao-manager\vendor\symfony\console\Application.php(223): Symfony\Component\Console\Application->doRunCommand(Object(Tenside\CoreBundle\Command\RunTaskCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 C:\development\contao\contao-manager\api\ApiApplication.php(71): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 C:\development\contao\contao-manager\vendor\symfony\console\Application.php(130): Contao\ManagerApi\ApiApplication->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 C:\development\contao\contao-manager\api\console(74): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}
--------------------------------------------------------
```

**Comment by bezin** [comment]
Should this better be posted in tenside/core?

**Comment by aschempp** [comment]
no, here's fine :)

**Comment by aschempp** [comment]
That's gonna be a real issue, as we cannot move an installation with absolute symlinks. The best solution I can think of is run `create-project --no-install`, then move the root files and run `composer install` afterwards…

**Comment by Toflar** [comment]
Should be fixed with 1.0 now 🎉
